tests/docker-compose-test.yaml

```diff
@@ -0,0 +1,53 @@
+version: '3.3'
+services:
+  integration_tests:
+    container_name: integration_tests
+    build: 
+      context: ./integration_tests
+    command: "true"
+    depends_on:
+      - manager
+      - agent
+    networks:
+      - test
+
+  manager:
+    container_name: manager
+    hostname: manager
+    build:
+      context: ../
+      dockerfile: ./tests/Dockerfile_manager
+    volumes:
+      - ${PWD}/tests/config.yaml:/cloud-manager/examples/config.yaml
```

Comment:
```suggestion
      - ${PWD}/config.yaml:/cloud-manager/examples/config.yaml
```

Reply:
same reason. `docker-compose up` is called from Makefile, and the path to `config.yaml` is set here regarding to Makefile location